adapters/limelightDigital/limelightDigital.go

```diff
@@ -0,0 +1,178 @@
+package limelightDigital
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/prebid/prebid-server/macros"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpointTemplate *template.Template
+}
+
+// Builder builds a new instance of the Limelight Digital adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	if config.Endpoint == "" {
+		return nil, errors.New("Endpoint  adapter parameter is not provided")
+	}
+	template, err := template.New("endpointTemplate").Parse(config.Endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse endpoint url template: %v", err)
+	}
+
+	bidder := &adapter{
+		endpointTemplate: template,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var requests []*adapters.RequestData
+	var errors []error
+
+	requestCopy := *request
+	for _, imp := range request.Imp {
+		limelightDigitalExt, err := getImpressionExt(&imp)
+		if err != nil {
+			return nil, append(errors, err)
+		}
+
+		url, err := buildEndpointURL(a.endpointTemplate, limelightDigitalExt)
+		if err != nil {
+			return nil, []error{err}
+		}
+
+		// Check if imp comes with bid floor amount defined in a foreign currency
+		if imp.BidFloor > 0 && imp.BidFloorCur != "" && strings.ToUpper(imp.BidFloorCur) != "USD" {
```

Comment:
please add test for this case

Reply:
Thanks for your comments. I added the test in https://github.com/prebid/prebid-server/pull/2539/commits/ef1e12ee84b4deedffd9d75323bf497629177837 commit. Could you check it and resolve the conversation?